src/Services/UserService.js

```diff
@@ -0,0 +1,87 @@
+import { rejects } from 'assert';
+import fs from 'fs/promises';
+import { resolve } from 'path';
+
+const getUsers = async (req, res) => {
+    try {
+        let readResult = await readUsers();
+
+        return readResult;
+    } catch (error) {
+        return error;
+    }
+};
+
+const createUsers = async (req, res) => {
```

Comment:
this function always overwrite users array instead of add new user to users array, please fix
request.body for this request  should be 
`{
    "login": "userdwqedwdwsdfsfsdw1",
    "password": "1sdsd234"
}`
 instead of 
```
{
    "users": [
        {
            "login": "userdwqedwdwsdfsfsdw1",
            "password": "1sdsd234"
        }
    ]
}
```